content/ja/docs/tasks/administer-cluster/dns-debugging-resolution.md

```diff
@@ -0,0 +1,291 @@
+---
+title:  DNS名前解決のデバッグ
+content_type: task
+min-kubernetes-server-version: v1.6
+---
+
+<!-- overview -->
+このページでは、DNSの問題を診断するためのヒントを提供します。
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}  
+クラスターは、CoreDNS {{< glossary_tooltip text="addon" term_id="addons" >}}またはその前身であるkube-dnsを使用するように構成する必要があります。
+
+{{% version-check %}}
+
+<!-- steps -->
+
+### テスト環境として使用するシンプルなPodを作成する {#create-a-simple-pod-to-use-as-a-test-environment}
+
+{{< codenew file="admin/dns/dnsutils.yaml" >}}
```

Comment:
Please copy `content/en/examples/admin/dns/dnsutils.yaml` to `content/ja/`, it caused build failure.